docs/content/preview/deploy/kubernetes/single-zone/oss/yugabyte-operator.md

```diff
@@ -12,32 +12,32 @@ menu:
 aliases:
   - /preview/deploy/kubernetes/yugabyte-operator/
   - /preview/deploy/kubernetes/oss/yugabyte-operator/
+  - /preview/deploy/kubernetes/operator-hub/
+  - /preview/deploy/kubernetes/oss/operator-hub/
+  - /preview/deploy/kubernetes/single-zone/oss/operator-hub/
 type: docs
 ---
 
-
 <ul class="nav nav-tabs-alt nav-tabs-yb">
   <li >
-    <a href="{{< relref "./helm-chart.md" >}}" class="nav-link">
+    <a href="../helm-chart/" class="nav-link">
       <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>
       Helm chart
     </a>
   </li>
   <li >
-    <a href="{{< relref "./yugabyte-operator.md" >}}" class="nav-link active">
-      <i class="fa-solid fa-cubes" aria-hidden="true"></i>
-      YugabyteDB operator
-    </a>
-  </li>
-  <li >
-    <a href="{{< relref "./operator-hub.md" >}}" class="nav-link">
+    <a href="../yugabyte-operator/" class="nav-link active">
       <i class="fa-solid fa-cubes" aria-hidden="true"></i>
-      Operator Hub
+      Kubernetes Operator (legacy)
     </a>
   </li>
 </ul>
 
+The Kubernetes Operator for YugabyteDB is no longer maintained. It is recommended that you use the actively maintained [helm charts](../helm-chart/) for YugabyteDB to install YugabyteDB on Kubernetes. For documentation on using the Kubernetes Operator for YugabyteDB, refer to the [GitHub repository](https://github.com/yugabyte/yugabyte-operator).
```

Comment:
```suggestion
The Kubernetes Operator for YugabyteDB is no longer maintained. It is recommended that you use the actively maintained [Helm charts](../helm-chart/) for YugabyteDB to install YugabyteDB on Kubernetes. For documentation on using the Kubernetes Operator for YugabyteDB, refer to the [GitHub repository](https://github.com/yugabyte/yugabyte-operator).
```

Because we are referring to the tool name.